app/views/home/contributors.md

```diff
@@ -0,0 +1,69 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that might be looking for ways to use those skills on an open source project.
+
+> Heading: A call to action. What is the one thing you want me to do? (3-6 words)
+
+# Make a contribution
+
+> Lead: Ok, you have my attention. Tell me more. Why should I care and what should I do about it? (~20ish words)
+
+Open source software is made by people just like you. This Friday, join the community and make one contribution.
+
+---
+
+> Why: this section should be the elevator pitch for why this person should want to get involved (5-10 words)
+
+## Get the most out of open source
+
+> Lead: What open source is and why I should care?
+
+Open source software is free for you to use and explore. Get involved to perfect your craft and be part of something big.
+
+> The deets: 3-4 of the most important reasons people participate
+
+- **Shape the future of software** - Your contributions help make technology better for everyone, developers and non-developers alike.
+- **Work with the best in the field** - Amazing developers participate in open source. Contribute to projects that change how software is built.
```

Comment:
```Contribute to software projects that run the world``` or some such? Bringing in my silicon valley vibe :trollface:

Also suggest ```Learn from the best in the field```, if it doesn't alter the meaning too much, to make it less intimidating. Even experienced developers are nervous about participating in open source BECAUSE they have to work with the best in the field.